pkg/csi_driver/node.go

```diff
@@ -76,7 +76,7 @@ func newNodeServer(driver *GCFSDriver, mounter mount.Interface, metaService meta
 		if err != nil {
 			return nil, err
 		}
-		lc, err := lockrelease.NewLockReleaseController(client, ns.features.FeatureLockRelease.Config)
+		lc, err := lockrelease.NewLockReleaseController(client, ns.features.FeatureLockRelease.Config, nil)
```

Comment:
Does the node driver still need a lock release controller? I think the node driver only need a k8s client when enabling lock release

Reply:
yeah, the node driver needs to use some config map utils from the lockReleaseController. Would be good to refactor the config map util methods so that node driver no longer relies on lock release controller. We should do that in a separate PR, WDYT?

Reply:
Added a TODO tag and created a bug to track b/375481562